source/opt/passes.cpp

```diff
@@ -39,7 +39,7 @@ namespace opt {
 
 bool StripDebugInfoPass::Process(ir::Module* module) {
   bool modified = !module->debugs().empty();
-  module->debugs().clear();
+  for (auto& inst : module->debugs()) inst.ToNop();
```

Comment:
No need anymore. They are all turned into Nop, which will be filtered at the end.